.github/workflows/test.yaml

```diff
@@ -231,6 +232,204 @@ jobs:
       matrix:
         python-version:
         - 3.7.10
+  build_wheels_linux:
+    container: quay.io/pypa/manylinux2014_x86_64:latest
+    name: Build wheels and fs_util (Linux)
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 10
+    - if: github.event_name == 'push'
+      name: Get commit message for branch builds
+      run: 'echo "COMMIT_MESSAGE<<EOF" >> $GITHUB_ENV
```

Comment:
YAML pro tip: Use the `|` operator for multi-line strings like this:

```
- if: github.event_name == 'push'
  name: Get commit message for branch builds
  run: |
    echo "COMMIT_MESSAGE<<EOF" >> $GITHUB_ENV
    echo "$(git log --format=%B -n 1 HEAD)" >> $GITHUB_ENV
    echo "EOF" >> $GITHUB_ENV
```

I realize this is generated by a script. Is there any way to get the output to be more idiomatic re multi-line strings?

Reply:
Not sure, but I think the more important thing to optimize more is idiomatic Python.